Packs/AHA/Integrations/AHA/AHA.py

```diff
@@ -0,0 +1,226 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+from typing import Dict
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+REPLACE = 'replace'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+URL_SUFFIX_PATTERN = f'/products/{REPLACE}/features/'
+EDIT_FIELDS = ['id', 'reference_num', 'name', 'description', 'workflow_status', 'created_at']
+DEFAULT_FIELDS = ['reference_num', 'name', 'id', 'created_at']
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    url = ''
+
+    def __init__(self,
+                 headers: dict,
+                 base_url: str,
+                 proxy: bool,
+                 verify: bool,
+                 url: str):
+        super().__init__(base_url=base_url, proxy=proxy, verify=verify, headers=headers)
+        self.url = url
+        self._headers['Content-Type'] = 'application/json'
+
+    def get_features(self,
+                     feature_name: str,
+                     fields: str,
+                     from_date: str,
+                     page: str,
+                     per_page: str) -> Dict:
```

Comment:
`per_page` isn't in?

Reply:
it is. but i get it from parameters passed to the command as a string and i pass it to the service API client as a string. so i thought no need in converting it to an integer